jsonb-generator/src/main/java/io/avaje/jsonb/generator/ComponentReader.java

```diff
@@ -61,12 +61,12 @@ private void readMetaData(TypeElement moduleType) {
       if (metaData != null) {
         metaData.value().stream()
           .map(TypeMirror::toString)
-          .forEach(componentMetaData::add);
+          .forEach(meta::add);
 
       } else if (metaDataFactory != null) {
```

Comment:
Looks like an existing bug here ... this ONLY adds factories if there isn't MetaData but there will commonly be both `@MetaData` and `@MetaData.JsonFactory`